workshop/README.md

```diff
@@ -0,0 +1,33 @@
+## Dataset description
```

Comment:
I think we need primarily some info on where this dataset is being used, if it's from Kaggle and structure is described there - no need to repeat much tbh.

Also, please try to wrap it at 80 like we do in the other projects - docs, blog, etc.